pkg/server/server.go

```diff
@@ -343,6 +354,10 @@ func NewServerWithOpts(opts ...OpenFGAServiceV1Option) (*Server, error) {
 		return nil, fmt.Errorf("request duration datastore count buckets must not be empty")
 	}
 
+	if len(s.requestDurationByDispatchCountHistogramBuckets) == 0 {
```

Comment:
Please add the corresponding test - similar to https://github.com/openfga/openfga/blob/5039e0d38af21e1179bc28b289f62bc9645150d9/pkg/server/server_test.go#L156

Reply:
Wait where is this test? 😆

Reply:
I pushed it to wrong branch 😢 - `rename-query-count-to-datastore-query-count`. Just cherry picked it to this one

Reply:
https://github.com/openfga/openfga/blob/4cbb08dfde9bdc9b3c7df1200e7b7b1a00b98a00/pkg/server/server_test.go#L168-L178